imap/message.c

```diff
@@ -69,6 +69,79 @@ static void imap_update_context (IMAP_DATA *idata, int oldmsgcount)
   }
 }
 
+char *imap_header_get_keywords(HEADER *h)
```

Comment:
shouldn't the return value be `const`? As I guess the returned value of this function is not meant to be mutated.